src/logs/views.py

```diff
@@ -108,6 +110,29 @@ def upload_multiple(request):
     return Response(serializer.data, status=status.HTTP_201_CREATED)
 
 
+@api_view(['GET'])
+def get_split_train_logs(request, pk):
+    value = Split.objects.get(pk=pk)
```

Comment:
Why this get has no try/catch, whereas the get_split_test_logs has a try/catch construct when retrieving the log?